gobblin-iceberg/src/main/java/org/apache/gobblin/iceberg/GobblinMCEProducer.java

```diff
@@ -198,6 +198,15 @@ private boolean verifyInput(Map<Path, Metrics> newFiles, List<String> oldFiles,
         }
         break;
       }
+      case change_property: {
+        if(newFiles != null || oldFiles != null) {
+          log.warn("{} new files and {} old files detected while no file alteration is performed",
```

Comment:
Since we always set new files to a dummy file, do you think we need to modify the condition for the warn log?